tests/Commands/DrupalCommandsTest.php

```diff
@@ -68,11 +68,61 @@ public function testPermissions(array $config, $command, $expected_error, $expec
         $this->assertEquals($expected_settings_file_permission, $siteSettingsPermissions);
     }
 
+    /**
+     * Test the services file setup.
+     *
+     * @param array $config
+     * @param array $data
+     * @param array $expected
+     *
+     * @dataProvider servicesSetupDataProvider
+     */
+    public function testServicesSetup(array $config, array $data, array $expected)
+    {
+        $configFile = $this->getSandboxFilepath('runner.yml');
+        file_put_contents($configFile, Yaml::dump($config));
```

Comment:
The code could be 
```
        file_put_contents($this->getSandboxFilepath('runner.yml'), Yaml::dump($config));
```

Reply:
This piece of code exists in a dozen places. If I change here I'll change al over.

Reply:
Fair enough.